src/commands/wait-in-queue.yml

```diff
@@ -191,6 +220,8 @@ steps:
         shopt -s nocasematch
         if [[ "$MESSAGE" == *'<< parameters.do_not_cancel_workflow_if_tag_in_commit >>'* ]]; then
           SKIP_COMMIT_ALLOWED=0
+          echo "Skip is disabled" 
+          echo "This Commit ($CIRCLE_SHA1) will not self-cancel and make $MAX_ATTEMPTS attempts to aquire the lock before failing"
```

Comment:
I'm a little confused by this. Do you mean:
```
This commit ($CIRCLE_SHA1) will not self-cancel or make any attempts to acquire the lock before failing
```
?

Reply:
yeah or maybe I should include "... at most $MAX_ATTEMPTS attempts to acquire the lock..."

Reply:
So it _will_ make N attempts, or it _wont_ make N attempts?

Reply:
The `not` plus `and` in the sentence is what's confusing me

Reply:
By not self-canceling the workflow has only one option which is to run through all iterations until it hits the MAX_ATTEMPTS limit. I'll just drop the max attempts part if it's confusing